### PR TITLE
Updating workflows/microbiome/nanopore-pre-processing from 0.1 to 0.2 

### PR DESCRIPTION
**Pull request by gxydevbot**
Hello! This is an automated update of the following workflow: **workflows/microbiome/nanopore-pre-processing**. I created this PR because I think one or more of the component tools are out of date, i.e. there is a newer version available on the ToolShed.

By comparing with the latest versions available on the ToolShed, it seems the following tools are outdated:
* `toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2/2.1.1+galaxy1` should be updated to `toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2/2.1.3+galaxy1`

The workflow release number has been updated from 0.1 to 0.2.

### CODE REVIEW

File: workflows/microbiome/nanopore-pre-processing/Nanopore-Pre-Processing.ga [hunk 1]
```diff
@@ -192,7 +192,7 @@
     ],
     "format-version": "0.1",
     "license": "MIT",
-    "release": "0.1",
+    "release": "0.2",
```
Comment (bebatut) [inline workflows/microbiome/nanopore-pre-processing/Nanopore-Pre-Processing.ga:195]:
```suggestion
    "release": "0.1.1",
```

File: workflows/microbiome/nanopore-pre-processing/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [0.2] 2024-06-28
```
Comment (bebatut) [inline workflows/microbiome/nanopore-pre-processing/CHANGELOG.md:3]:
```suggestion
## [0.1.1] 2024-06-28
```

File: workflows/microbiome/nanopore-pre-processing/Nanopore-Pre-Processing.ga [hunk 3]
```diff
@@ -192,7 +192,7 @@
     ],
     "format-version": "0.1",
     "license": "MIT",
-    "release": "0.1",
+    "release": "0.2.1",
```
Comment (bebatut) [inline workflows/microbiome/nanopore-pre-processing/Nanopore-Pre-Processing.ga:195]:
```suggestion
    "release": "0.1.1",
```

File: workflows/microbiome/nanopore-pre-processing/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,10 @@
...
+
+### Automatic update
+- `toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2/2.1.1+galaxy1` was updated to `toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2/2.1.3+galaxy1`
```
Comment (bebatut) [inline workflows/microbiome/nanopore-pre-processing/CHANGELOG.md:6]:
```suggestion
- `toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2/2.1.1+galaxy1` was updated to `toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2/2.1.3+galaxy1`
- `toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.74+galaxy0` was update to `toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.74+galaxy1`
```

**Review by mvdbeek — APPROVED** [review]

**Comment by lldelisle** [comment]
@bebatut should I delete the branch to get the new updates?

**Pull request closed without merging**